src/__stories__/list-story.tsx

```diff
@@ -147,6 +150,27 @@ const Template: StoryComponent<Args & {boxed?: boolean}> = ({
                     onPress,
                 };
                 break;
+            case 'iconButton':
+                controlProps = {
+                    iconButton: {
+                        Icon: IconPlayFilled,
+                        onPress: () => console.log('Pressed button'),
+                        type: 'brand',
+                        backgroundType: 'solid',
+                    },
+                };
+                break;
+            case 'iconButton and onPress':
+                controlProps = {
+                    iconButton: {
+                        Icon: IconPlayFilled,
```

Comment:
```suggestion
                        Icon: IconTrashCanRegular,
```

To let the play/pause icon for toggle example